LandLord-api/src/main/java/biz/princeps/landlord/api/IMaterialsManager.java

```diff
@@ -20,7 +20,9 @@ public interface IMaterialsManager {
     Material getLongGrass();
 
     ItemStack getPlayerHead(UUID owner);
-    ItemStack getPlayerHead(OfflinePlayer owner);
+    ItemStack getPlayerHead(UUID owner, String texture);
+    ItemStack getPlayerHead(OfflinePlayer owner, String texture);
+    ItemStack getPlayerHead(String texture);
```

Comment:
this is mainly about keeping compatibility with latest, without reinventing the wheel. I mainly went for a "make it work approach" instead of producing fine code.

Reply:
Actually this is a huge mess right now. legacy might not even work anymore right now...
This is a big abstraction issue, since legacy requires the texture as base64 encoded whatever and latest requires the url encoded in the base46 string

Reply:
The latest way might not even work on older paper versions between 1.13 and 1.18 I think. once again I hate multiversion support xD

Reply:
Yes, for proper multi-version support the old strategy must exist as fallback on versions that don't have the bukkit `PlayerProfile` API. I think it's fine to extract the url only as needed. Or alternatively pass an object that holds both the base64 string and the url instead of just a string - that would also help with proper typing.